docs/reference/languagedetails.md

```diff
@@ -78,7 +78,15 @@ future.
 
 ### Characters
 
-In the future, Dark will support individual characters.
+Characters in Dark are single display characters. Technically, they are Extended
```

Comment:
"single display characters" isn't clear to me.

Perhaps, "a character in Dark represents a character that you see on your screen, such as ...  . This is in contrast to most languages, where a character represents a byte, or a Unicode codepoint. The technical name for a Dark character is an EGC, and can best be thought of as supporting the entire character you see in front of you - in the case of emoji, this includes such things as combining characters and skin tones.